sources/rest_api/config_setup.py

```diff
@@ -403,16 +450,30 @@ def create_response_hooks(
     the default behavior is to raise an HTTP error.
 
     Example:
+        def set_encoding(response, *args, **kwargs):
+            response.encoding = 'windows-1252'
+            return response
+
+        def remove_field(response: Response, *args, **kwargs) -> Response:
+            payload = response.json()
+            for record in payload:
+                del record["email"]
+            modified_content: bytes = json.dumps(payload).encode("utf-8")
+            response._content = modified_content
+            return response
```

Comment:
I find it a bit cumbersome to unpack the response manually – the detector does it so well and now the user has to do it again by hand. 

Maybe users would prefer a different API for the use cases of adding or deleting fields. Instead of operating on the raw response I'd prefer to operate on the parsed records.